test/__snapshots__/marketplaces.test.ts.snap

```diff
@@ -227,7 +237,18 @@ AmazonMarketplace {
   "currency": "EGP",
   "id": "ARBP9OOSHTCHU",
   "name": "Egypt",
+  "sellingPartner": Object {
```

Comment:
Do we know this for sure, that Egypt is part of `eu-west-1`?

Reply:
I found them in "[Selling Partner API endpoints](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/developer-guide/SellingPartnerApiDeveloperGuide.md#selling-partner-api-endpoints)"